vmaas_sync/pkg_sync.go

```diff
@@ -0,0 +1,95 @@
+package vmaas_sync //nolint:golint,stylecheck
+
+import (
+	"app/base"
+	"app/base/database"
+	"app/base/models"
+	"app/base/utils"
+	"github.com/RedHatInsights/patchman-clients/vmaas"
+	"github.com/antihax/optional"
+	"github.com/jinzhu/gorm"
+	"github.com/pkg/errors"
+)
+
+func syncPackages(tx *gorm.DB, pkgs []string) error {
```

Comment:
Have you tested it with real vmaas instance? I've got `time="2020-07-27T15:49:59Z" level=error msg="manual called syncing failed" err="Storing packages: Get packages: 400 Bad Request"` after trying to sync packages during perf-testing. I am not able to do perf-testing without synced packages.

Reply:
Fixes implemented in https://github.com/RedHatInsights/patchman-engine/pull/295